### PR TITLE
Fixing issue #2097

### PR DESCRIPTION
**Pull request by kkvvii84**
Removing non bind variables in SQL calls

### CODE REVIEW

File: library/calendar.inc [hunk 1]
```diff
@@ -15,7 +15,7 @@ function getNextAppointment($pid)
 //this function returns calendar data given date and provider
 function getCalendarDay($datetime, $owner, $groupname)
 {
-    $query = "select * from calendar where time like '$datetime%' and owner like '$owner' and groupname like '$groupname' order by time";
+    $query = "select * from calendar where time like ? and owner like ? and groupname like ? order by time", array($datetime, $owner, $groupname);
```
Comment (bradymiller) [inline library/calendar.inc]: just minor fix to:
```
$query = "select * from calendar where time like ? and owner like ? and groupname like ? order by time", array($datetime.'%', $owner, $groupname);
```

File: library/calendar.inc [hunk 2]
```diff
@@ -151,7 +151,7 @@ function getCalendarRanges($fromdate, $todate, $username)
 //get calendar dates in a range of dates
 function getCalendarbyTimeandRange($time, $fromdate, $todate, $username)
 {
-    $query = "select * from calendar where time like '% $time%' and time>='$fromdate' and time<'$todate' and owner like '$username' order by time";
+    $query = "select * from calendar where time like ? and time>= ? and time< ? and owner like ? order by time", array($time, $fromdate, $todate, $username);
```
Comment (bradymiller) [inline library/calendar.inc]: just minor fix to:
```
$query = "select * from calendar where time like ? and time>= ? and time< ? and owner like ? order by time", array('% '.$time.'%', $fromdate, $todate, $username);
```

**Comment by bradymiller** [comment]
hi @kkvvii84 ,
Couple a couple minor fixes for you to address, and then this will be ready for codebase. thanks! -brady

File: library/calendar.inc (hunk 2 above)
Comment (kkvvii84) [reply library/calendar.inc]: Updated as suggested.

File: library/calendar.inc (hunk 1 above)
Comment (kkvvii84) [reply library/calendar.inc]: Updated as suggested.

**Comment by bradymiller** [comment]
hi @kkvvii84 ,
Code looks good and bringing into codebase. thanks for the contribution!
-brady

**Pull request merged**